apps/judicial-system/web/src/routes/Court/Indictments/Subpoena/Subpoena.tsx

```diff
@@ -74,6 +77,9 @@ const Subpoena: FC = () => {
       const allDefendantsUpdated = await Promise.all(promises)
 
       if (!allDefendantsUpdated.every((result) => result)) {
+        toast.error(formatMessage(errors.updateDefendant))
```

Comment:
Note that a toast is already displayed if the update failes. `useCourtArrangements` uses `setAndSendToServer` from `useCase` which used the generated `useUpdateCase` hook. A failed update there displays a taost.

Please remove this toast so we do not get a duplicate toast when something goes wrong.